clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -504,18 +505,67 @@ static Value *EmitISOVolatileStore(CodeGenFunction &CGF, const CallExpr *E) {
   return Store;
 }
 
+static CallInst *CreateBuiltinCallWithAttr(CodeGenFunction &CGF, StringRef Name,
+                                           llvm::Function *FPBuiltinF,
+                                           ArrayRef<Value *> Args,
+                                           unsigned ID) {
+  llvm::CallInst *CI = CGF.Builder.CreateCall(FPBuiltinF, Args);
+  llvm::AttributeList AttrList;
+  // sincos() doesn't return a value, but it still has a type associated with
+  // it that corresponds to the operand type.
+  CGF.CGM.getFPAccuracyFuncAttributes(
```

Comment:
Why do we need a AttrList here? Is there a scenario where more than one such attribute can be attached to a builtin call?

Thanks

Reply:
No, I can't think of one.

Reply:
Actually I think you are right that the AttrList can be a single attribute instead of a list. Added a TOTO comment as we discussed offline. Thanks.